src/firecfg/firecfg.config

```diff
@@ -481,6 +481,7 @@ lowriter
 # lrzip - disable until we fix CLI archivers for makepkg on Arch (see discussion in #3095)
 # lrztar - disable until we fix CLI archivers for makepkg on Arch (see discussion in #3095)
 # lrzuntar - disable until we fix CLI archivers for makepkg on Arch (see discussion in #3095)
+luarocks
```

Comment:
Needs discussion: Do we want to firecfg build-systems/package-managers by default? (related: #4519)